refinery/selenium_testing/tests.py

```diff
@@ -26,6 +29,10 @@ class SeleniumTestBase(StaticLiveServerTestCase):
 
     def setUp(self, site_login=True, initialize_guest=True,
               public_group_needed=True):
+
+        # Start a pyvirtualdisplay for geckodriver to interact with
+        self.display = Display(visible=0, size=(1900, 1080))
```

Comment:
We did have this before, See here: https://github.com/refinery-platform/refinery-platform/pull/1650/files/c733de5aea8aa4871c925bf8cd17bf705758b71b#diff-f4bed93e08ec88c8c01e26197a799d9bL17

I've moved this into the `setUp()` and the stopping of said virtual display into `tearDown()`